app/models/document.rb

```diff
@@ -46,6 +54,51 @@ def facet_options(facet)
     finder_schema.options_for(facet)
   end
 
+  def format_specific_metadata
```

Comment:
I could put it in a `Wiring` file? :trollface: 

For real it is. I'll add some explanation as to what it does.